pages/members/index.tsx

```diff
@@ -57,11 +60,15 @@ const HonoraryMembersPage: NextPage = () => {
       </Accordion>
     </Stack>
   );
+  const [honorSelected, setHonorSelected] = React.useState(true);
+  const [orderSlected, setOrderSlected] = React.useState(false);
```

Comment:
I think the best way to do this use use the location as the "state", then you can use `useRouter` to check which "tab" you are on and just replace the router location when clicking on the buttons.

Reply:
I.e. have a regex or string comparison on `router.pathname` and store that in a variable. Then, when changing the path with `router.push` or `router.replace`, the component would update without the need for any state (due to the useRouter hook).